index.js

```diff
@@ -74,7 +82,8 @@ const pseudoLocalization = (() => {
     }
   ) => {
     opts.blacklistedNodeNames = options.blacklistedNodeNames;
-    pseudoLocalize(document.body, options);
+    opts.strategy = options.strategy;
+    pseudoLocalize(document.body);
```

Comment:
No more need to pass the `options` here? And what do you do with `ops` (assuming it's relevant to this PR)?

Reply:
Correct. Since the closure contains all options already, no need to both consume them directly and pass them to a function inside the same closure.